core/src/main/java/com/netflix/conductor/core/execution/WorkflowSweeper.java

```diff
@@ -14,10 +14,20 @@
  * limitations under the License.
  */
 /**
- * 
+ *
  */
 package com.netflix.conductor.core.execution;
 
+import com.netflix.conductor.core.WorkflowContext;
+import com.netflix.conductor.core.config.Configuration;
```

Comment:
Does this file have any change related to `executed` flag ?

Reply:
no changes, just some code cleanup